Mixpanel/MPNotificationViewController.m

```diff
@@ -101,7 +103,21 @@ - (void)viewDidLoad
 
         if (self.notification.callToAction && [self.notification.callToAction length] > 0) {
             [self.okayButton setTitle:self.notification.callToAction forState:UIControlStateNormal];
-            [self.okayButton sizeToFit];
+        }
+        
+        if (self.notification.style != nil) {
+            if ([self.notification.style isEqualToString:@"light"]) {
```

Comment:
Seems like we don't need to both of these `if` statements. the second one should suffice.